run/GCHP/ExtData.rc.templates/ExtData.rc.carbon

```diff
@@ -298,7 +362,10 @@ SIB_BBIO_CO2 kg/m2/s N Y F%y4-%m2-%d2T%h2:00:00 none none CO2 ./HcoDir/CO2/v2022
 CO2_NET_TERRESTRIAL kg/m2/s N Y - none none CO2 ./HcoDir/CO2/v2022-11/BIO/Net_terrestrial_exch_5.29Pg.generic.1x1.nc
 
 # --- CO2 Ship Emissions ---
-CEDS_CO2_SHP kg/m2/s N Y F%y4-%m2-01T00:00:00 none none CO2_shp ./HcoDir/CEDS/v2021-06/%y4/CO2-em-anthro_CMIP_CEDS_%y4.nc
+CEDS_CO2_SHP kg/m2/s N Y F%y4-%m2-01T00:00:00 none none CO2_shp ./HcoDir/CEDS/v2024-06/%y4/CEDS_CO2_0.1x0.1_%y4.nc
+#
+# ICOADS_SHIP is off by default
+#ICOADS_SHIP_CO N Y F2002-%m2-01T00:00:00 none none CO ./HcoDir/ICOADS_SHIP/v2014-07/ICOADS.generic.1x1.nc
```

Comment:
Same comment as above. I think these lines can be removed.

Reply:
Now fixed in 6de9ca5